applications/rag/frontend/main.tf

```diff
@@ -48,13 +92,13 @@ resource "kubernetes_service" "rag_frontend_service" {
     selector = {
       app = "rag-frontend"
     }
-    session_affinity = "ClientIP"
+    session_affinity = "None"
     port {
       port        = 8080
       target_port = 8080
     }
 
-    type = "ClusterIP"
+    type = "NodePort"
```

Comment:
can we have a flag to allow ClusterIP + port forward?

Reply:
Will do, discussed offline, we will highly recommend the IAP